config/validate/util/highlight.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2015 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"fmt"
+	"strings"
+)
+
+func Highlight(src []byte, offset int64) (int, int, string) {
+	lineNum := 1
+	colNum := 1
+	var startOfLine int64
+	var i int64
+	newline := false
+	for i = 0; i < offset; i++ {
+		if newline {
+			newline = false
+			lineNum++
+			colNum = 1
+			startOfLine = i
+		}
+		if src[i] == '\n' {
+			newline = true
+		}
+		colNum++
+	}
```

Comment:
Could use a combination of `bytes.Count()` and `bytes.LastIndex()` here on a subslice, no? Slightly less efficient, but this is on the error path anyway.